examples/doc-scan/src/main/resources/application.yml.example

```
@@ -0,0 +1,15 @@
+server:
+  port: 8443
+  ssl:
+    enabled: true
+    key-store: "server.keystore.jks"
+    key-store-password: javaexample
+  servlet:
+    session:
+      persistent: true
+      timeout: 360000
+com:
+  yoti:
+    docscan:
+      sdkId: "your-client-sdk-id-from-yoti-hub"
+      pemFileLocation: "your-pem-file-from-yoti-hub"
```

Sorry being pedantic, but this isn't a location, it's specifying a filename, right? If we put a filepath in or something it'll break won't it?
```suggestion
      pemFilename: "your-pem-file-from-yoti-hub"
```